### PR TITLE
Hostpath driver 1.0 YAML update

### PR DESCRIPTION
**Pull request by vladimirvivien**
Updates documentation to point to YAML configs for 1.0.0 update.

### CODE REVIEW

File: book/src/Example.md [hunk 1]
```diff
@@ -91,7 +93,7 @@ The CSI sidecar apps are going to connect to the CSI driver, therefore
 starting it first helps avoid timeouts and intermittent container
 restarts:
 
-> $ kubectl create -f [https://raw.githubusercontent.com/kubernetes/kubernetes/f40a5d1155aae95105a4e9bb8933d750c666e350/test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpathplugin.yaml](https://github.com/kubernetes/kubernetes/blob/f40a5d1155aae95105a4e9bb8933d750c666e350/test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpathplugin.yaml)
+> $ kubectl create -f [https://github.com/kubernetes/kubernetes/blob/1a3fbf1f13a28f3102edc43ab2dca95471453cd8/test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpathplugin.yaml](https://github.com/kubernetes/kubernetes/blob/1a3fbf1f13a28f3102edc43ab2dca95471453cd8/test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpathplugin.yaml)
```
Comment (msau42) [inline book/src/Example.md]: Instead of pulling from kubernetes e2e test manifests, should we pull from yamls in kubernetes-csi/drivers?

**Comment by pohly** [comment]
The top of the README.md states that the example is known to work for 1.12. Before changing something on that page, please decide how people will find working examples for the different supported Kubernetes versions.

File: book/src/Example.md (hunk 1 above)
Comment (pohly) [reply book/src/Example.md]: That makes sense, but someone would first have to update the drivers repo such that it actually has up-to-date yaml files. Right now there aren't any.

Comment (msau42) [reply book/src/Example.md]: Let's do that first, so we don't have to update this twice?

File: book/src/Setup.md [hunk 2]
```diff
@@ -171,7 +171,7 @@ To configure your driver using the registrar sidecar, you can configure the cont
         fieldRef:
           apiVersion: v1
           fieldPath: spec.nodeName
-    image: quay.io/k8scsi/driver-registrar:v0.2.0
+    image: quay.io/k8scsi/driver-registrar:v1.0.0
```
Comment (saad-ali) [inline book/src/Setup.md]: Move this to 1.0.1
1.0.0 has a bug that will prevent it from working (it advertises the wrong supported version).

Comment (pohly) [reply book/src/Setup.md]: And replace driver-registrar with node-driver-registrar.

**Comment by vladimirvivien** [comment]
Closing because this has been fixed elsewhere.

**Pull request closed without merging**